github/props.conf

```diff
@@ -35,57 +41,151 @@ NO_BINARY_CHECK = true
 SHOULD_LINEMERGE = false
 category = Metrics
 description = Collectd daemon format. Uses the write_http plugin to send metrics data to a Splunk platform data input via the HTTP Event Collector.
-disabled = false
 pulldown_type = 1
 
 [github_json]
-FIELDALIAS-dependabot = "alert.affected_package_name" AS affected_package_name "alert.external_identifier" AS cve "alert.external_reference" AS url "alert.most_recent_instance.location.path" AS alert_location_path "alert.rule.description" AS alert_description "alert.rule.security_severity_level" AS severity_level "alert.severity" AS severity eventtype AS vendor_product "repository.html_url" AS dest "repository.owner.login" AS user
-EVAL-dvc = replace(host, ":\d+", "")
-EVAL-signature = CASE(isnull(alert_description), UPPER(severity) + " Dependency Vulnerability on package " + affected_package_name, 1=1, alert_description)
-EVAL-xref = if(isnotnull(affected_package_name), affected_package_name, alert_location_path)
-FIELDALIAS-RepoAlias = "organization.login" ASNEW organization "repository.name" ASNEW repository_name
-EVAL-category = if(isnotnull(alert_description), "code", if(isnotnull(affected_package_name), "dependency", ""))
-disabled = false
-pullrequest_base_sha =
-EVAL-pullrequest_base_sha = 'pull_request.base.sha'
-EVAL-pullrequest_base_user_login = 'pull_request.base.user.login'
-EVAL-repository_name = 'repository.name'
+# Basic settings
+TRUNCATE = 100000
 KV_MODE = json
-EXTRACT-commit_hash = | spath commits{} output=commits  | mvexpand commits  | rex field=commits "(?<=\"id\"\:\")(?<commit_hash>\w*)"
+pulldown_type = true
+DATETIME_CONFIG =
```

Comment:
Again thank you for doing all the wrangling to pull this work from dev into the repo!